aspnetcore/mvc/views/razor.md

```diff
@@ -567,6 +567,20 @@ C# Razor keywords need to be double escaped with `@(@C# Razor Keyword)`, for exa
 * namespace
 * class
 
+## View compilation
+
+Razor views are compiled at runtime when the view is invoked. If you prefer to compile your Razor views and deploy them with your app follow these steps:
+
+1. Add a reference to "Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design" under the "dependencies" section.
+2. Add a reference to "Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Tools" under the "tools" section.
+3. Add a postpublish script to invoke view compiler:
+
+```
```

Comment:
Maybe add the word "project.json" somewhere to ensure it turns up in searches when we look for affected articles.